doc/user_guide/mva.rst

```diff
@@ -297,6 +302,42 @@ of components to keep. Setting this to a small number is recommended to keep
 the computation time small. Often it is useful to run a PCA decomposition first
 and use the scree plot to determine a value for "output_dimension".
 
+Robust non-negative matrix factorization
+-----------------------------------
+
+In a similar manner to the online, robust methods that complement PCA above,
+HyperSpy includes an online robust NMF method. This is based on the OPGD (Online
+Proximal Gradient Descent) algorithm of :ref:`[Zhao2016] <Zhao2016>`.
+
+Note that this requires "output_dimension" to be specified.
+
+.. code-block:: python
+
+   >>> s.decomposition(algorithm='ORNMF',
+                       output_dimension=3)
+
+As with ORPCA described above, the MomentumSGD method  is useful for scenarios
```

Comment:
the Momentum Stochastic Gradient Descent (Momentum SGD) method...

Reply:
good idea

Reply:
I've defined MomentumSGD higher up now.